tests/ucan/ucan.node.test.ts

```diff
@@ -0,0 +1,90 @@
+import expect from "expect"
+import { webcrypto } from "one-webcrypto"
+
+import * as base64 from "../../src/common/base64.js"
+import * as did from "../../src/did/index.js"
+import * as keystore from "../../src/keystore.js"
+import * as ucan from "../../src/ucan/index.js"
+
+
+describe("UCAN", () => {
+
+  it("can be built", async function() {
+    const u = await ucan.build({
+      audience: await randomRsaDid(),
+      issuer: await did.ucan()
+    })
+
+    expect(await ucan.isValid(u)).toBe(true)
+  })
+
+  // TODO: Can't figure out why this doesn't work.
+  //       This gives a localforage error → "No available storage method found"
+  //
+  // it("can validate a UCAN with a valid proof", async function() {
```

Comment:
Can anyone figure out why this doesn't work? See TODO comment. I don't see where this calls localforage 🤔

Reply:
I suspect this may be because `localforage` won't have a storage option in the node environment. 

We could work around this by dependency-injecting an in-memory store if the code were in webnative, but `localforage` is used in `keystore-idb` in this case.

At one point, we talked about implementing a dependency-injectable version of the keystore, with the idea that we could have an in-memory version to use for tests. Not sure if it's worth the effort, but it would be nice to have and could probably get us better test coverage in general.

Another option might be adding a DI storage implementation to `keystore-idb`.

Reply:
I think this should already work. There's some code in `setup/node.ts` that tries to make the keystore work in-memory instead of on localforage. Does that perhaps need to be imported here?

Reply:
Ah right I have to change the `.store` from the keystores, I'll use the in-memory rsa store from `setup/node` 👍